src/lib/Microsoft.Health.Fhir.Ingest/Service/MeasurementEventNormalizationService.cs

```diff
@@ -45,7 +46,7 @@ public class MeasurementEventNormalizationService : IDataNormalizationService<Ev
             Data.IConverter<EventData, JToken> converter,
             IExceptionTelemetryProcessor exceptionTelemetryProcessor,
             int maxParallelism,
-            int asyncCollectorBatchSize = 200)
+            int asyncCollectorBatchSize = 25)
```

Comment:
Do we have any data on the impact of this change?

Reply:
No hard numbers. I had originally put in `200` as a starting point, though. With the intent on tweaking it as needed. I can revert this change.